stockflux-securities-master/src/components/SecuritiesTable.css

```diff
@@ -51,7 +51,7 @@
 
 .table-deleting-spinner-container,
 .securities-message-container {
-  margin-top: var(--margin-large);
+  margin: var(--margin-small) 0px;
```

Comment:
Just a tip: You can omit the units (`px` in this case) if you want to set the size of something to 0

Reply:
okay